package.json

```diff
@@ -94,5 +93,16 @@
     "@storybook/types": ">=7.5.3",
     "react": ">=16.9.0",
     "react-dom": ">=16.9.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
Useless?

Reply:
I went with Storybook's config, basically it's saying that
> This is a set of config values that will be used at publish-time. It's especially handy if you want to set the tag, registry or access, so that you can ensure that a given package is not tagged with "latest", published to the global public registry or that a scoped module is private by default.

We shouldn't have issue with it

Reply:
Yeah but you only set the access to publish which is the default. But NVM it's not very important

Reply:
oh got ya